spring-cloud-sleuth-instrumentation/src/main/java/org/springframework/cloud/sleuth/instrument/kafka/TracingKafkaPropagatorGetter.java

```diff
@@ -30,7 +30,7 @@
  *
  * @author Anders Clausen
  * @author Flaviu Muresan
- * @since 3.0.3
+ * @since 3.1.0
```

Comment:
Why would this change?

Reply:
Another PR for this functionality was initially raised against master (#1906), and @marcingrzejszczak suggested that we raise it against 3.1.x instead (which I did in #1920 ). However when #1920 was merged, we missed bumping up the `since` version, therefore doing it in this PR (see https://github.com/spring-cloud/spring-cloud-sleuth/pull/1920#issuecomment-824897747)